digitalpublicgoods/neonatal-nutrition-digital-accelerator-kit-project.json

```diff
@@ -0,0 +1,94 @@
+{
+  "name": "NEONATAL NUTRITION DIGITAL ACCELERATOR KIT PROJECT",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://www.digitalhealthatlas.org/en/-/projects/5111/published"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "Installation manual - https://github.com/IntelliSOFT-Consulting/human-milk-bank/blob/main/README.md",
+      "User manual - https://docs.google.com/document/d/1dBImVDN82pDT-Z4Nb2h4iCu34Itz0TpbcE9Q1U25oAs/edit",
+      "Admin manual - https://docs.google.com/document/d/155XLs7bFn1VMM-3ix-ZjfbZ3Rih51wo43s-0NMLJkvM/edit",
+      "Software Requirements Specification - https://docs.google.com/document/d/15lryinyblKPD3E6L-jTbdepox230qaBWR6phHM-AnT0/edit?pli=1"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "Data can be extracted via a web portal export functionality which provides the data in JSON and PDFs formats."
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "The Kenya Data Protection Act 2019"
+    ],
+    "adherenceSteps": [
+      "Privacy policy - https://docs.google.com/document/d/14Fpd95JgZJPC2H6HFjeOZppGjJg6Z_SU/edit"
```

Comment:
I would suggest product owners to put this documentation in the repo (Markdown o PDF) or in the website if possible.